flathub.json

```
@@ -0,0 +1,5 @@
+{
+    "automerge-flathubbot-prs": false,
+    "disable-external-data-checker": false,
+    "require-important-update": false
+}
```

```suggestion
{
    "disable-external-data-checker": false,
    "require-important-update": false
}
```

Please don't use automerge we will block it soon